service/models.py

```diff
@@ -112,8 +113,7 @@ class Wishlist(db.Model):
     app = None
 
     def __repr__(self):
-        return "<Wishlist %r user_id=[%s] items[%s]>" % \
```

Comment:
This fails the pep8 guideline - check 
 flake8 --count --max-complexity=10 --statistics model, service

Same for others